core/unit_test/TestExecutionSpace.hpp

```diff
@@ -36,6 +36,16 @@ TEST(TEST_CATEGORY, execution_space_as_class_data_member) {
 }
 #endif
 
+TEST(TEST_CATEGORY, execution_space_moved_from) {
+  TEST_EXECSPACE exec;
+  TEST_EXECSPACE other = std::move(exec);
+  // NOLINTNEXTLINE(bugprone-use-after-move,-warnings-as-errors)
```

Comment:
Why did you include `,-warnings-as-errors)`
What does that even do

Reply:
Just an oversight from copying from the compiler putput verbatim.